<launch>
	<node pkg="x52_joyext" type="x52_value2buttonColor_node" name="Color_Button_0" output="screen"> 
		<param name="input_type" value="11"/>
		<param name="joy_axis_button" value="4"/>
		<param name="joy_axis_or_button" value="false"/>
		<param name="color_led" value="4" />	
		<param name="setup_string" value="G|-0.5|Y|0.5|R"/>
		<remap from="/Color_Button_0/in" to="/joy" />
		<remap from="/Color_Button_0/led" to="/x52/leds" />
	</node>
		
	<node pkg="x52_joyext" type="x52_value2buttonColor_node" name="Color_Button_1" output="screen"> 

		<param name="input_type" value="11"/>
		<param name="joy_axis_button" value="4"/>
		<param name="joy_axis_or_button" value="false"/>
		<param name="color_led" value="2" />
		<param name="setup_string" value="G|-0.8|Y|-0.5|R|0.5|Y|0.8|G" />
		<remap from="~/in" to="/joy" />
		<remap from="~/led" to="/x52/leds" />
	</node>
	
	
	
	<!-- Button Example -->
	<node pkg="x52_joyext" type="x52_value2buttonColor_node" name="Color_Button_2" output="screen"> 
		<param name="input_type" value="11"/>
		<param name="joy_axis_or_button" value="true"/>
		<param name="joy_axis_button" value="1"/>
		<param name="color_led" value="0" />
		<param name="setup_string" value="O|0.5|G|1" />
		<remap from="~/in" to="/joy" />
		<remap from="~/led" to="/x52/leds" />
	</node>
</launch>